Providers/Profile.swift

```diff
@@ -717,10 +672,6 @@ open class BrowserProfile: Profile {
         fileprivate var syncTimer: Timer?
 
         fileprivate var backgrounded: Bool = true
-        public func applicationDidEnterBackground() {
-            self.backgrounded = true
-            self.endTimedSyncs()
-        }
```

Comment:
was this method never called or using `endTimedSyncs` in deinit is enough?

Reply:
I did a quick check and it does seem like its never called 🙄

Reply:
I was also surprised by this one.